File: java/0125-valid-palindrome/Main.java
public class Main {
    public static void main(String[] args) {
        Solution solution = new Solution();

        System.out.println("true: " + solution.isPalindrome("A man, a plan, a canal: Panama"));
        System.out.println("false: " + solution.isPalindrome("race a car"));
        System.out.println("true: " + solution.isPalindrome(" "));
    }
}

File: c++/0125-valid-palindrome/main.cpp
#include <cstdlib>
#include <iostream>
#include <string>

#include "solution.cpp"

using namespace std;

int main() {
    Solution solution;

    string answer1 = solution.isPalindrome("A man, a plan, a canal: Panama") ? "true" : "false";
    cout << "true: " << answer1 << endl;

    string answer2 = solution.isPalindrome("race a car") ? "true" : "false";
    cout << "false: " << answer2 << endl;

    string answer3 = solution.isPalindrome(" ") ? "true" : "false";
    cout << "true: " << answer3 << endl;

    return EXIT_SUCCESS;
}
